sormas-backend/src/test/java/de/symeda/sormas/backend/contact/ContactFacadeEjbTest.java

```diff
@@ -980,6 +981,17 @@ public void testGetContactCountsByCasesForDashboard() {
 		assertThat(result[2], equalTo(1));
 	}
 
+	@Test
+	public void testGetContactCountsByCasesForDashboardWithHugeList() {
+
```

Comment:
Please amend the existing unit test for that method. A new test also puts several seconds of additional execution time that can be avoided.